product_docs/docs/pem/9/certificates/index.mdx

```diff
@@ -0,0 +1,313 @@
+
+1. Copy new `root.crt`, `server.key`, and `server.crt` to the data directory of the backend database server:
+
+   ```shell
+   cp root.crt server.key server.crt /var/lib/edb/as<x>/data
+   ```
+
+1. Change the owner and permissions of new certificates and key files same as the data directory:
+
+   ```shell
+   cd /var/lib/edb/as<x>/data/
+   chown enterprisedb server.* root.crt ca_certificate.crt
+   chmod 600 server.* root.crt ca_certificate.crt
+   ```
+
+    
+   !!! Note
+       Do not restart the PEM server now, or else all the registered agents will stop working.
+
+1. Replace each PEM agent SSL certificates with the trusted third-party CA certificates. For more information, see this [section](/pem/latest/certificates/#replace-the-agent-self-signed-ssl-certificates-with-the-trusted-third-party-ca-certificates).
+
+1. Restart the PEM server.
+
+   - On Linux:
+
+   ```shell
+   # Running as root
+   systemctl start edb-as-<x>
+   ```
+
+   Restarting the backend database server, restarts the PEM server.
+
+### Replace the agent self-signed SSL certificates with the trusted third-party CA certificates
+
+1. Use psql to find all the agent identifiers(IDs) we need to replace the SSL certificates:
+
+   ```shell
+   psql -U enterprisedb -d pem --no-psqlrc -t -A -c "SELECT id FROM pem.agent WHERE active=true"
+   ```
+
+1. After identifying the agents that need key files, generate an `agent<ID>.key` for each agent. To generate the key, execute the command:
+
+   ```shell
+   openssl genrsa -out agent<ID>.key 4096 
+   ```
+   
+   Where `<ID>` is the agent identifier.
+
+1. Generate a Certificate Signing Request (CSR) for each agent:
+
+   ```shell
+   openssl req -new -key agent<ID>.key -out agent<ID>.csr -subj '/C=IN/ST=MH/L=Pune/O=PEM/CN=agent<ID>'
+   ```
+   
+   Where you replace `<ID>` in `CN` with appropriate agent identifier.
+
+1. Ask your CA to sign the CSR and generate agent certificate for you. 
+
+1. Copy the certificate and key files on respective hosts, where `<ID>` matches the `agent_id` in the `/usr/edb/pem/agent/etc/agent.cfg` file.
+
+1. Change the ownership and permission on the new `agent<ID>.crt` and `agent<ID>.key` file:
+
+    ```shell
+    chown root agent<ID>.crt agent<ID>.key
+    chmod 600 agent<ID>.crt agent<ID>.key
+    ```
+
+1. Take backup of old agent certificate and key file:
+
+   ```shell
+   # Running as root
+   mkdir root/.pem/certs
+   mv root/.pem/agent<ID>.* root/.pem/certs
+   ```
+
+1. Replace each agent's certificate and key file with the newly generated files: 
+   
+   ```shell
+   cp agent<ID>.key agent<ID>.crt root/.pem
+   ```
+
+7. Restart the PEM agent service.
+
+   - On Linux:
+
+   ```shell
+   # Running as root
+   systemctl restart pemagent
+   ```
+
+   - On Windows:
+     You can use the Services applet to restart the PEM agent. The PEM agent service is named Postgres Enterprise Manager Agent. Select the service name in the Services dialog box and select **Restart the service**.
```

Comment:
I suggest adding a troubleshooting/test section at the end which covers the following two steps (should be referenced appropriately from instructions above).

- How to validate a certificate against root.crt
- How to test ssl connection from an agent host using psql

Reply:
I have added this in [762a97f](https://github.com/EnterpriseDB/docs/pull/3627/commits/762a97f07ff8074de4fa36a336336bddaa904324) so I'll close this conversation